### PR TITLE
feat(pdf): parse disableFontFace option from viewer options 

### PR DESCRIPTION
**Pull request by mickr**
Allow parsing of the disableFontFace viewer option

### CODE REVIEW

**Comment by boxcla** [comment]
Verified that @mickr has signed the CLA. Thanks for the pull request!

**Comment by mickr** [comment]
> "Revert index.html" failed?

Nah, it just ran prettier on the index so I am leaving it.

File: README.md [hunk 1]
```diff
@@ -131,25 +131,27 @@ preview.show(fileId, accessToken, {
...
+| viewers                          | {}            | Maps options to a valid viewer type **Use these at your own risk**                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                |
```
Comment (jstoffan) [inline README.md]: I would remove this "use at your own risk" and leave it on the `disableFontFace` option, instead.

File: README.md [hunk 2]
```diff
@@ -131,25 +131,27 @@ preview.show(fileId, accessToken, {
...
+| viewers.Document.disableFontFace | false         | By default fonts are converted to OpenType fonts and loaded via font face rules. If disabled, fonts will be rendered using a built-in font renderer that constructs the glyphs with primitive path commands. **Use this option at your own risk**                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                 |
```
Comment (jstoffan) [inline README.md]: We should probably also add `viewers.Presentation.disableFontFace`, as that option will also work.

**Pull request merged**